readme.md

```diff
@@ -1,32 +1,86 @@
 # Microsoft Azure IoT SDKs for Java
 This repository contains the following:
-* **Azure IoT Hub Device SDK**: to connect client devices to Azure IoT Hub (supports Java 7+ and Android API 17+)
-* **Azure IoT Hub Service SDK**: enables developing back-end applications for Azure IoT (supports Java 7+)
+* **Azure IoT Hub device SDK for Java**: to connect client devices to Azure IoT Hub (supports Java 7+ and Android API 17+)
+* **Azure IoT Hub service SDK for Java**: enables developing back-end applications for Azure IoT (supports Java 7+)
+
+The API reference documentation for the device SDK is [here][java-api-reference-device].
+
+The API reference documentation for the service SDK is [here][java-api-reference-service].
 
 To find SDKs in other languages for Azure IoT, please refer o the [azure-iot-sdks][azure-iot-sdks] repository
 
 ## Developing applications for Azure IoT
 Visit [Azure IoT Dev Center](http://azure.com/iotdev) to learn more about developing applications for Azure IoT.
 
 ## How to use the Azure IoT SDKs for Java
+
 Devices and data sources in an IoT solution can range from a simple network-connected sensor to a powerful, standalone computing device. Devices may have limited processing capability, memory, communication bandwidth, and communication protocol support. The IoT device SDKs enable you to implement client applications for a wide variety of devices.
-The API reference documentation for the device SDK is [here][java-api-reference-device].
-The API reference documentation for the service SDK is [here][java-api-reference-service].
 * On Linux and Windows:
    * **Using Maven**: the simplest way to use the Azure IoT SDKs for Java to develop apps is to leverage Maven packages:
       * [Device SDK][device-maven]
       * [Service SDK][service-maven]
+   * **Clone the repository**: The repository is using [GitHub Submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules) for its dependencies. In order to automatically clone these submodules, you need to use the --recursive option as described here:
+   ```
+    git clone --recursive https://github.com/Azure/azure-iot-sdk-java.git  
+   ```
```

Comment:
Java does not use submodule and we should not be. Why was this needed ?

Reply:
Updated